node/src/remote/index.ts

```diff
@@ -66,8 +68,60 @@ export class RemoteConnection implements Connection {
     }
   }
 
-  async createTable<T> (name: string | CreateTableOptions<T>, data?: Array<Record<string, unknown>>, optsOrEmbedding?: WriteOptions | EmbeddingFunction<T>, opt?: WriteOptions): Promise<Table<T>> {
-    throw new Error('Not implemented')
+  async createTable<T> (nameOrOpts: string | CreateTableOptions<T>, data?: Array<Record<string, unknown>>, optsOrEmbedding?: WriteOptions | EmbeddingFunction<T>, opt?: WriteOptions): Promise<Table<T>> {
```

Comment:
It doesn't look like you handle `writeOptions` (either as `opt` or `optsOrEmbedding`) at all.  Is this intentional?

Reply:
this is meant to support create by overwrite. Since we can overwrite by insert already, I just skipped this for now.